core/src/main/java/io/github/tcdl/MsbContext.java

```diff
@@ -89,8 +112,11 @@ public MsbContextBuilder withDefaultChannelMonitorAgent(boolean withDefaultChann
         }
         
         /**
-         * Create MsbContext and initialize it with Config from reference.conf
-         * @throws MsbException if an error happens during initialization  
+         * Create {@link MsbContext} and initialize it with configuration from reference.conf(property file inside MSB library)
```

Comment:
It's doesn't my line,but I will replace it with "configuration from reference.conf(property file inside MSB library) or application.conf, which will override library properties"